starboard/BUILD.gn

```diff
@@ -86,6 +86,9 @@ group("gn_all") {
       }
     }
   }
+  if (sb_crashpad_integration) {
+    deps += [ "//third_party/crashpad/client" ]
```

Comment:
Do you plan to use this interface directly in the xb1 port, instead of going through https://github.com/youtube/cobalt/blob/main/third_party/crashpad/wrapper/wrapper.h?

Reply:
Yes, this time I included the client directly, as purpose of this wrapper is not clear to me yet. And the switch `sb_crashpad_integration` is rather a temporary solution.